### DISCUSSION TITLE
How to return to the previous version of the log?

### DISCUSSION THREAD

**Discussion by MikleBas** [magicbug/Cloudlog#2506] [General]
The current release is, to put it mildly, unsuccessful (crude)
in addition to the shortcomings I found, another one appeared today (maybe it’s just me, I don’t know), the prompts stopped working when entering a call sign in the window, before it was shown as a new qso or a repeated one - but now even this is gone, there are no qrz.com and hamqth - this is already annoying and you have to reload the page...

![4](https://github.com/magicbug/Cloudlog/assets/99086477/7f522ea2-6562-4dd7-b010-314e32abf0cf)

**Comment by magicbug** [comment]
Think we need to keep language polite and just because you having issues doesn't mean a release is crude, I've well over 100+ copies running the latest stable without issue.

As with any online software when upgrading you take a full backup before carrying it out which then allows you to roll back, in fact its stated in the documentation.

Usually when Country is marked as "None" its because the Country Files aren't being maintained regularly, for Callsign field no longer colouring itself it sounds like your browsers cached the CSS/JS files and it needs cleared.

**Reply by MikleBas** [reply]
Hello!
Sorry friend, I didn’t mean to offend anyone, I apologize!
What command should be given in the server console to roll back?
Your product is very good, probably all my difficulties are due to possible blockages in our country, perhaps..

**Reply by magicbug** [reply]
You meant what you wrote until challenged.

All previous versions of tagged Cloudlog are available at https://github.com/magicbug/Cloudlog/releases only current release are supported.
